brage-import/src/main/java/no/sikt/nva/brage/migration/lambda/BrageEntryEventConsumer.java

```diff
@@ -79,6 +80,7 @@ public class BrageEntryEventConsumer implements RequestHandler<S3Event, Publicat
     public static final String AGGREGATION = "aggregation";
     public static final String NONE = "none";
     public static final String ISBN = "isbn";
+    public static final int MAX_ACCEPTABLE_LEVENSHTEIN_DISTANCE = 10;
```

Comment:
This means that we have to remove max 10 chars from string to achieve to similar string